docs/whats-new/zowe-v3-frequently-asked-questions.md

```diff
@@ -0,0 +1,123 @@
+
+2. Are **[what kind of?]** libraries updated to the current latest versions?
+
+In Zowe V3, the Application Framework uses Angular 16 and Webpack 5, which are the latest current releases.**[are there other libraries that aren't up-to-date, and if so, should we list them here?]**
+
+3. What React versions are supported by Zowe Desktop?
+
+It is possible for Desktop extensions to use different versions of React. While the Desktop is not built on React, it is still compatible with extensions that import React.
+
+4. Is the ZEN application running on Zowe Desktop or on z/OS?
+
+ZEN runs on a laptop/desktop and connects to z/OS via standard methods.
+
+5. Is it possible to validate and change the `zowe.yaml` file and job definition within ZEN?
+
+Yes.
+
+6. What happens if in ZEN you click on the SMP/E? **[what is smp/e here? is it a button? a drop-down option? how do users get to smp/e?]**
+
+You are guided through the SMP/E installation steps.
+
+7. What is the connection between Zowe Store and ZEN?
+
+Zowe Store is separate from ZEN, but they do share **[some commands? all commands?]** `zwe` commands.
+
+8. Does the Zowe Store handle Zowe upgrades?
+
+No, it offers server side extensions.
+
+### Zowe System Installation and Configuration
+
+1. Does Zowe provide a Software Bill of Materials (SBOM)?
+
+   Zowe V3 provides SBOMs in an SPDX format. **[have there been changes to the format? if so, what are the changes?]**
+
+## User questions
+
+### Zowe API Mediation Layer
+
+1. Will Zowe V3 APIML support clients with onboarding enabler V2?
+
+   The API ML in Zowe V3 supports clients with onboarding enablers from Zowe V2. It also works the other way around: The onboarding enablers from Zowe V3 will continue working with Zowe V2 API ML.
+
+2. The jump to Java 17 means that we have to maintain two separate versions of our application, one with apiml-enabler V2 and one with V3, to support customers who want to stay with Java 8. Do you have a recommendation or a workaround for supporting both Java versions?
+
+   The plain Java Enabler from Zowe V2 works in Zowe V3. You can keep this until Zowe V4.
+
+   Zowe V4 will only support Spring Boot, which requires Java 17. We recommend building applications for Java 17 to ensure that applications continue to be compatible with API ML.
+
+3. Can you speak about the migration from Zuul to Spring Cloud Gateway? Today there are two separate gateway services in API ML with separate config.
+
+   We expect to support only Spring Cloud Gateway. The spring-cloud-gateway configuration should move under the component.gateway namespace. We expect to have **[should users be doing the config here?]** one Spring Cloud Gateway cluster ahead **[ahead or between?]** of multiple sysplexes and then one cluster on every sysplex. Most of the configuration that was used should remain, including the new one for Spring Cloud Gateway.
```

Comment:
@balhar-jakub - please respond

Reply:
The users shouldn't need to do any specific configuration. 
Ahead of multiple sysplexes. THe diagram is here - https://docs.zowe.org/stable/user-guide/api-mediation/api-mediation-multi-tenancy#overview-of-central-and-domain-api-mls

Reply:
Follow-up question: What do we mean by "two separate services"? Are we talking about running both zuul and spring cloud gateway, or are we talking about a configuration w/ a central API ML gateway _and_ multiple domain API ML gateways? The current answer discusses multiple domain gateways behind the central gateway, but it's not clear how this relates back to the FAQ question. Please advise.

Reply:
We talk about a cluster of central Gateways. In V2 it's always Spring Cloud Gateway with configuration that makes the cluster behave as central. 

I am a bit confused. Original question was about migrating between Zuul and Spring Cloud Gateway. 
This question has two parts answer. 

For domain gateways there should be no change. The properties will remain in gateway namespace. 
For Central gateway there is a change as a result, the properties move from cloud-gateway to gateway namespace.

Reply:
Hey Jakub, We have a few questions/comments after discussing with the team in Pitt, can you take a look?

> For domain gateways there should be no change.

Is it correct that for v3, Spring Cloud Gateway will be used for domain gateways without any configuration change? 

> For Central gateway there is a change as a result, the properties move from cloud-gateway to gateway namespace.

Does this mean that config which was previously under `cloud-gateway` in `zowe.yaml` that they config will now be under `gateway`? 

> We expect to support only Spring Cloud Gateway. The spring-cloud-gateway configuration should move under the component.gateway namespace. We expect to have **[should users be doing the config here?]** one Spring Cloud Gateway cluster ahead **[ahead or between?]** of multiple sysplexes and then one cluster on every sysplex. Most of the configuration that was used should remain, including the new one for Spring Cloud Gateway.

For this, we should probably remove "should" in favor of something more definitive unless we truly don't know the outcome.  For example, "... configuration should move ... " to "...configuration **will** move ..."

Thanks!

Reply:
I believe that now we can move to more definitive answer, this was move of questions from Office Hours in January. 

Spring Cloud Gateway will be used for domain gateways without any configuration change. 
Spring Cloud Gateway in central will move to gateway as it's the same component in a different configuration.